editor-components/feature/feature1/feature1.tsx

```diff
@@ -1,162 +1,540 @@
 import * as React from "react";
-import { BaseFeature } from "../../EditorComponent";
+import { BaseFeature, TypeUsableComponentProps } from "../../EditorComponent";
 import styles from "./feature1.module.scss";
 import { ComposerIcon } from "../../../composer-base-components/icon/icon";
+import ComposerLink from "../../../../custom-hooks/composer-base-components/Link/link";
 
-type ICircleBoxes = {
+type CardComponent = {
+  image: string;
+  date: string;
+  read_time:string;
   title: string;
   description: string;
-  icon: string;
+  profile_image: string;
+  fullname: string;
+  written: string;
+  url: string;
 };
+
+type MiniCardComponent = {
+  date2: string;
+  read_time2:string;
+  title2: string;
+  description2: string;
+  profile_image2: string;
+  fullname2: string;
+  written2: string;
+  url2: string;
+};
+
+type Button = {
+    buttonText: string;
+    url: string;
+    icon: string;
+};
+
+
 class Feature1 extends BaseFeature {
   constructor(props?: any) {
     super(props, styles);
-
     this.addProp({
       type: "array",
-      key: "circle-boxes-content",
-      displayer: "Circle Boxes Content",
+      key: "card",
+      displayer: "Card",
       value: [
         {
           type: "object",
-          key: "circle-boxes",
-          displayer: "Circle Boxes",
+          key: "card_info",
+          displayer: "Card Info",
           value: [
+            {
+              type: "image",
+              key: "image",
+              displayer: "Image",
+              value: "https://tushi-nextjs.vercel.app/_next/image?url=%2Fimages%2Fblog%2F08.jpg&w=384&q=75",

```

Comment:
Images need to be uploaded to the Spica storage, and the values must be retrieved from the storage URL.